cmd/kubetoken/kubetoken.go

```diff
@@ -17,20 +17,25 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
-	"github.com/okteto/okteto/pkg/types"
 	"os"
 
+	"github.com/okteto/okteto/pkg/okteto"
+	"github.com/okteto/okteto/pkg/types"
+
 	contextCMD "github.com/okteto/okteto/cmd/context"
 
-	"github.com/okteto/okteto/pkg/errors"
-	"github.com/okteto/okteto/pkg/okteto"
 	"github.com/spf13/cobra"
 )
 
 type Serializer struct {
 	KubeToken types.KubeTokenResponse
 }
 
+// oktetoClientProvider provides an okteto client ready to use or fail
```

Comment:
do we still need `oktetoClientProvider` in types? it seems unused now?

Reply:
Yes, We are using it in the validation to check that the okteto client has the kubetoken API endpoint

Reply:
I mean `types.oktetoClientProvider`